packages/now-static-build/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@now/static-build",
-  "version": "0.12.3-canary.0",
+  "version": "2.0.0-canary.0",
```

Comment:
@AndyBitz @leo Do you remember why `@now/static-build` and `@now/next` were not bumped to 3?

Reply:
Ok I see that both of those builders return more than one lambda. I guess the follow up PR will needs some special cases for these two builders in order to pin them to the major version 2 (when all others will be 3 or possible 4 very soon).